src/components/leaderboard/leaderboard-tab/LeaderboardTab.tsx

```diff
@@ -48,13 +49,26 @@ interface LeaderboardTabProps {
   activePanelKey?: number;
 }
 
-export interface LeaderboardItem {
+export interface TabItem {
   rank?: number;
+  id: number;
   name: string; // for now, these are assumed to be unique
   rightSide: React.ReactNode;
   collapseContent?: React.ReactNode;
 }
 
+export const leaderboardItemsToTabItems = (
```

Comment:
Should this function be exported? I don't think it should be used outside of this component.